core/orchestrator.go

```diff
@@ -581,6 +582,10 @@ func (n *LivepeerNode) transcodeSeg(config transcodeConfig, seg *stream.HLSSegme
 				string(md.ManifestID), md.AuthToken.SessionId, seg.SeqNo, len(tSegments[i].Data))
 			return terr(fmt.Errorf("ZeroSegments"))
 		}
+		if md.CalcPerceptualHash && tSegments[i].PHash == nil {
```

Comment:
I think we should return an error here because if O doesn't receive a phash from T that means that a phash will be missing in the response that O sends to B and B will reject O's response. WDYT?